static/src/js/components/ProjectPanels/ProjectPanels.js

```diff
@@ -332,10 +333,50 @@ class ProjectPanels extends PureComponent {
 
       const {
         title,
-        isCSDA: collectionIsCSDA
+        isCSDA: collectionIsCSDA,
+        cloudHosted,
+        duplicateCollections = []
       } = collectionMetadata
 
-      const { [collectionId]: collectionDataQualitySummaries = [] } = dataQualitySummaries
+      let { [collectionId]: collectionDataQualitySummaries = [] } = dataQualitySummaries
+
+      const hasDataQualitySummary = collectionDataQualitySummaries.length > 0
+      const hasDuplicateCollection = duplicateCollections.length > 0
+
+      const dataQualityHeader = (() => {
+        if (hasDataQualitySummary && hasDuplicateCollection) {
+          return 'Important data quality and availability information'
+        }
+        if (hasDataQualitySummary) {
+          return 'Important data quality information'
+        }
+        if (hasDuplicateCollection) {
+          return 'Important data availability information'
+        }
+        return ''
+      })()
+
+      if (hasDuplicateCollection) {
+        const duplicateCollectionId = duplicateCollections[0]
+        collectionDataQualitySummaries = [...collectionDataQualitySummaries, {
+          id: 'duplicate-collection',
+          summary: cloudHosted
+            ? (
+              <>
+                <span>This dataset is hosted in the cloud. The dataset is also </span>
+                <Link to={`/search?q=${duplicateCollectionId}`}>hosted in a NASA datacenter</Link>
```

Comment:
These links need to use PortalLinkContainer and not this Link component. PortalLinkContainer will handle adjusting the url for a portal if the user is viewing a portal besides the default portal

Reply:
Take a look at the Back to Search link on the project page for a good example (found in SecondaryToolbar.js), it keeps the current search params active when it returns to the search page.

I think since we have the conceptId of the duplicate collection, we should link them to the collection page directly, so the path would be /search/granules with a `p` parameter that includes the new conceptId to set it as the focusedCollection. So the full URL it links to would look something like this:
```
/search/granules?p=C2208418228-POCLOUD!C1972954180-PODAAC&pg[0][v]=f&pg[0][gsk]=-start_date&sp[0]=-75.98285%2C38.74822&tl=1655419723.226!3!!&lat=30.118590972093784&long=-130.82660293579102
```

Reply:
done